pkg/controller/common/helpers.go

```diff
@@ -1159,7 +1159,6 @@ func (n namespacedEventRecorder) Eventf(object runtime.Object, eventtype, reason
 func (n namespacedEventRecorder) AnnotatedEventf(object runtime.Object, annotations map[string]string, eventtype, reason, messageFmt string, args ...interface{}) {
 	n.delegate.AnnotatedEventf(ensureEventNamespace(object), annotations, eventtype, reason, messageFmt, args...)
 }
-
```

Comment:
Nit: it looks like there is a line removed here, was there a reason for that? If not could you reset this? Thanks!